src/main.rs

```diff
@@ -88,9 +88,11 @@ fn app() -> Result<(), Error> {
                 mailgun::run(&token, &encryption_key, &team_api, dry_run)?;
             }
             "zulip" => {
+                let username = std::env::var("ZULIP_USERNAME")
+                    .with_context(|_| "failed to get the ZULIP_API_TOKEN environment variable")?;
```

Comment:
Nit: wrong environment variable name here :sweat_smile: 

At this point it might be worth just doing an `env("ZULIP_USERNAME")?` helper that adds the right context to the std error.